integration-sqs-to-lambda/Function.go

```diff
@@ -1,13 +1,12 @@
 package integration_sqs_to_lambda
 
 import (
-	"context"
 	"fmt"
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
 )
 
-func handler(ctx context.Context, event events.SQSEvent) error {
+func handler(event T) error {
```

Comment:
My original comment wasn't clear. The T was just a place holder for types in the function signature. 
This handler will need to replace T with events.SQSEvent to compile correctly. It should look like:
`func handler(event events.SQSEvent) error {`

Reply:
refactored code